scripts/generate-json-schema.ts

```
@@ -192,6 +192,16 @@ async function writeFile(path: string, content: string) {
 	return attempt();
 }
 
+async function readFile(path: string): Promise<Result.Result<string, any>> {
+	return Result.try({
+		try: async () => {
+			const file = Bun.file(path);
+			return file.text();
+		},
+		catch: error => error,
+	})();
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)